mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Requests tests
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*'  }
+
+  Scenario: Given an item Id, a user Id, and a pickup location, attempt to create a page request when the applicable request policy disallows pages
+    * def itemId = call uuid1
+    * def servicePointId = call uuid1
+    * def userId = call uuid1
+    * def userBarcode = 'FAT-1030UBC'
+    * def itemBarcode = 'FAT-1030IBC'
+    * def groupId = call uuid1
+    * def instanceId = call uuid1
+    * def locationId = call uuid1
+    * def holdingId = call uuid1
+    * def materialTypeId = call uuid1
+    * def materialTypeName = 'book'
+    * def extRequestTypes = ["Hold", "Recall"]
+
+    # post a material type and policies
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostMaterialType') { extMaterialTypeId: #(materialTypeId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPolicies') { extRequestTypes: #(extRequestTypes) }
+
+    # post an item
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostServicePoint')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostLocation')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(itemId), extItemBarcode: #(itemBarcode) }
+
+    # post a group and an user
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup')
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(userId), extUserBarcode: #(userBarcode) }
+
+    # post a request and verify that the user are not allowed to create a page request
+    * def requestId = call uuid1
+    * def requestEntityRequest = read('classpath:vega/mod-circulation/features/samples/request-entity-request.json')
+    * requestEntityRequest.requesterId = userId
+    * requestEntityRequest.requestType = 'Page'
+    Given path 'circulation', 'requests'
+    And request requestEntityRequest
+    When method POST
+    Then status 422
+    * def error = response.errors[0]
+    And match error.message == 'Page requests are not allowed for this patron and item combination'
```

Comment:
```suggestion
    And match $.errors[0].message == 'Page requests are not allowed for this patron and item combination'
```
This variable is used only once, I guess we don't really need it

Reply:
done